Online/Entity/OnlineCreature.cs

```diff
@@ -277,6 +277,51 @@ public void SuckedIntoShortCut(IntVector2 entrancePos, bool carriedByOther)
             enteringShortCut = false;
         }
 
+        public void BroadcastTookFlight(AbstractRoomNode.Type type, WorldCoordinate start, WorldCoordinate dest)
+        {
+            if (currentlyJoinedResource is RoomSession room)
+            {
+                RainMeadow.Debug(this);
+                if (id.type == 0) throw new InvalidProgrammerException("here");
```

Comment:
Intentional error message in the exception or residual from testing?

Reply:
copypasted from suckedintoshortcut